### PR TITLE
Make bow craftable

### PR DESCRIPTION
**Pull request by This1sPanda**
## About The Pull Request

-Added Bow recipes in roguecrafting/items.dm

## Why It's Good For The Game

As of now, bows seemed to be completely absent from any recipes file, meaning it was impossible to get your hands on one if you didn't spawn with it, despite the arrows for it being as cheap as dirt. Opens up a bit the range game to none range starting classes and allows for a cheap but accesible way to fight crossbow spam in the PvP areas.

### CODE REVIEW

**Comment by This1sPanda** [comment]
Originally I was thinking the same, especially when looking at the rope sprite (which seems more like a climbing rope then any bow string) In fact that's probably a better idea then rope to use 3 fibers in hindsight.

But the log has two uses. First a logical one, implying the wood of the bow is carved from one entire log and not sticks stacked together (which fits the bow sprite more, unless I get to separate both the bow and the makeshift bow down the line). And the second was a balance issue. Bows are still a solid range option, and if well trained, on par with the mid to high tier weapons. Making it as easy to craft as a stone knife wouldn't feel right. Like that, it's still cheap but ask more ressources then doing a stone axe or wooden swords.

**Comment by This1sPanda** [comment]
Sure ! That sounds logical, I'll modify the PR a bit for that and it should be good.

File: code/modules/roguetown/roguecrafting/items.dm [hunk 1]
```diff
@@ -231,3 +231,9 @@
 
 /obj/item/rogueweapon/shovel/small/crafted
 	sellprice = 5
+
+/datum/crafting_recipe/roguetown/bow
```
Comment (ChungusGamer666) [inline code/modules/roguetown/roguecrafting/items.dm:235]: making a bow should be a delicate process, please increase skilldiff to 2

**Review by ChungusGamer666 — CHANGES_REQUESTED** [review]
:)

**Comment by Real-MAGNUM** [comment]
Waiting for changes+conflict resolution

**Comment by ChungusGamer666** [comment]
Goin on one week without the conflicts being resolved nor changes made

**Pull request closed without merging**